Sources/Clappr_iOS/Classes/Plugin/Core/MediaControl/PlayButton.swift

```diff
@@ -1,21 +1,16 @@
 open class PlayButton: MediaControl.Element {
-    open class override var name: String {
-        return "PlayButton"
-    }
-
-    override open var panel: MediaControlPanel {
-        return .center
-    }
+    open class override var name: String { "PlayButton" }
 
-    override open var position: MediaControlPosition {
-        return .center
-    }
+    override open var panel: MediaControlPanel { .center}
+    override open var position: MediaControlPosition { .center }
 
     public var playIcon = UIImage.fromName("play", for: PlayButton.self)!
     public var pauseIcon = UIImage.fromName("pause", for: PlayButton.self)!
 
-    public var button: UIButton! {
+    public var button: UIButton? {
```

Comment:
I think this way can be a good idea. 

```suggestion
    public var button: UIButton = {
        let view = UIButton(type: .custom)
        view.accessibilityIdentifier = "PlayPauseButton"
        view.setImage(playIcon, for: .normal)
        view.imageView?.contentMode = .scaleAspectFit
        
        return view
    }()
```

Reply:
Thank you. We have to assign the button inside the view from `MediaControl.Element` and we create the button instance on `Element.render()` to conform with the architecture. 
What do you think?

Reply:
OK! LGTM